Assets/Scripts/WackyTappy/SlimeSquishedGameController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SlimeSquishedGameController : MonoBehaviour
+{
+
+ 
+    public static int numOfSquishedSlimes;
+    private int numOfSlimes = 5;
```

Comment:
I'd rename this to `_numOfSlimesToWin`, since that way we're denoting that it's a private variable, as well as that it's the number we're trying to reach, basing the win condition off that number.